packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -7,6 +7,10 @@ import { stdJson } from "forge-std/StdJson.sol";
 import { Executables } from "scripts/Executables.sol";
 import { Chains } from "scripts/Chains.sol";
 
+// Global constant for the `useFaultProofs` slot in the DeployConfig contract, which can be overridden in the testing
+// environment.
+bytes32 constant USE_FAULT_PROOFS_SLOT = bytes32(uint256(61));
```

Comment:
It'll be more maintainable to add a `DeployConfig` function that returns the compiler provided slot offset, rather than hardcode it here. Though I imagine we only need the override temporarily so this is OK.

Reply:
Can't do this, we need the slot prior to the `DeployConfig`'s deployment to perform the override unfortunately.

Reply:
Bummer. Could Deploy a dummy `DeployConfig` to get the info 😛. Probably worse though.